src/types/relationships.ts

```diff
@@ -0,0 +1,43 @@
+import { RelationshipFromIntegration } from '@jupiterone/jupiter-managed-integration-sdk';
+
+export interface StandardizedOktaUserGroupRelationship
```

Comment:
More types we shouldn't need ultimately, the converter tests should be expressing these requirements from the conversion functions.

Reply:
Removed